graylog2-web-interface/src/components/event-definitions/event-definitions/EventDefinitionsContainer.tsx

```diff
@@ -0,0 +1,152 @@
+  },
+  status: {
+    renderCell: (eventDefinition) => <StatusCell status={eventDefinition?.scheduler?.is_scheduled} />,
+    staticWidth: 100,
+  },
+  prority: {
+    staticWidth: 50,
+  },
+});
+
+const EventDefinitionsContainer = () => {
+  const [visibleColumns, setVisibleColumns] = useState(INITIAL_COLUMNS);
+  const paginationQueryParameter = usePaginationQueryParameter(undefined, 20);
+  const [searchParams, setSearchParams] = useState<SearchParams>({
+    page: paginationQueryParameter.page,
+    pageSize: paginationQueryParameter.pageSize,
+    query: '',
+    sort: {
+      attributeId: 'title',
+      direction: 'asc',
+    },
+  });
+  const { data: paginatedEventDefinitions, refetch: refetchEventDefinitions } = useEventDefinitions(searchParams);
+  const columnRenderers = customColumnRenderers();
+  const columnDefinitions = useMemo(
+    () => ([...(paginatedEventDefinitions?.attributes ?? []), ...CUSTOM_COLUMN_DEFINITIONS]),
+    [paginatedEventDefinitions?.attributes],
+  );
+
+  const onPageChange = useCallback(
+    (newPage: number, newPageSize: number) => setSearchParams((cur) => ({ ...cur, page: newPage, pageSize: newPageSize })),
+    [],
+  );
+
+  const onSearch = useCallback((newQuery: string) => {
+    paginationQueryParameter.resetPage();
+    setSearchParams((cur) => ({ ...cur, query: newQuery }));
+  }, [paginationQueryParameter]);
+
+  const onReset = useCallback(() => {
+    onSearch('');
+  }, [onSearch]);
+
+  const onColumnsChange = useCallback((newVisibleColumns: Array<string>) => {
+    setVisibleColumns(newVisibleColumns);
+  }, []);
+
+  const onSortChange = useCallback((newSort: Sort) => {
+    setSearchParams((cur) => ({ ...cur, sort: newSort, page: 1 }));
+  }, []);
+
+  const renderEventDefinitionActions = useCallback((listItem: EventDefinition) => (
+    <EventDefinitionActions eventDefinition={listItem} refetchEventDefinitions={refetchEventDefinitions} />
+  ), [refetchEventDefinitions]);
+
+  const renderBulkActions = (
+    selectedEventDefinitionsIds: Array<string>,
+    setSelectedEventDefinitionsIds: (eventDefinitionsId: Array<string>) => void,
+  ) => (
+    <BulkActions selectedDefintionsIds={selectedEventDefinitionsIds}
+                 setSelectedEventDefinitionsIds={setSelectedEventDefinitionsIds} />
+  );
+
+  if (!paginatedEventDefinitions) {
+    return <Spinner text="Loading Event Definitions information..." />;
+  }
+
+  const { elements, pagination: { total } } = paginatedEventDefinitions;
+
+  return (
+    <PaginatedList onChange={onPageChange}
+                   pageSize={searchParams.pageSize}
+                   totalItems={total}>
+      <div style={{ marginBottom: 5 }}>
+        <SearchForm onSearch={onSearch}
+                    onReset={onReset}
+                    queryHelpComponent={<QueryHelper entityName="stream" />} />
+      </div>
+      <div>
+        {elements?.length === 0 ? (
+          <NoSearchResult>No Event Definition have been found</NoSearchResult>
```

Comment:
```suggestion
          <NoSearchResult>No Event Definition has been found</NoSearchResult>
```

or

```suggestion
          <NoSearchResult>No Event Definitions have been found</NoSearchResult>
```

Also, event definitions should be lowercase, but we could argue that for us it is a "Feature Name", also product-wise and thus keep it like that.